test/common/schemaform/save-load-actions.unit.spec.js

```diff
@@ -185,10 +190,10 @@ describe('Schemaform save / load actions:', () => {
       global.fetch.returns(Promise.reject(new Error('No network connection')));
 
       thunk(dispatch).then(() => {
-        done(new Error("Should not call the dispatch's .then() on a network failure"));
```

Comment:
It doesn't really make sense after I took out the unit test specific promise rejections. We really care that the failure status is set (tested in the `then` here) and that we don't get an unhandled/unexpected rejection (tested by rejecting in the `catch` here).